src/bin/playground.cpp

```diff
@@ -2,9 +2,18 @@
 
 #include "types.hpp"
 
+#include "operators/print.hpp"
+#include "table_generator.hpp"
+
 using namespace opossum;  // NOLINT
 
 int main() {
-  std::cout << "Hello world!!" << std::endl;
+  TableGenerator generator;
```

Comment:
?

Reply:
this was for the presentation, reverted it